lib/Fmt_ast.ml

```diff
@@ -3132,6 +3154,7 @@ and fmt_case c ctx ~first ~last ~padding case =
       , ( Exp {pexp_desc= Pexp_function _ | Pexp_match _ | Pexp_try _; _}
         , (Pexp_match _ | Pexp_try _) ) ) ->
         2
+    | _, (_, Pexp_beginend _) -> 2
```

Comment:
I think you indented to match for pattern matches inside the beginend ? This applies to any beginend.

Reply:
done